routes/rooms.go

```diff
@@ -93,28 +94,28 @@ func CreateEmergencyRoom(ctx *gin.Context) {
 }
 
 // UpdateEmergencyRoom godoc
-// @Summary     update an emergency room by id
-// @Tags 		emergency-rooms
-// @Produce 	json
-// @Param       authorization header    string              true    "Bearer: <TOKEN>"
-// @Param       id   		path 		string				true	"Emergency Room's ID"
-// @Success     200  		{object} 	GetSingleERResponse
-// @Failure     501  		{object}	HTTPErrorResponse
-// @Router      /er/{id}	[patch]
+// @Summary    update an emergency room by id
+// @Tags       emergency-rooms
+// @Produce    json
+// @Param      authorization                   header      string                true    "Bearer: <TOKEN>"
+// @Param      id                              path        string                true    "Emergency Room's ID"
+// @Success    200                             {object}    GetSingleERResponse
+// @Failure    501                             {object}    HTTPErrorResponse
+// @Router     /er/{id}                        [patch]
 func UpdateEmergencyRoom(ctx *gin.Context) {
 	_ = ctx.Param("id")
 	SendError(ctx, http.StatusNotImplemented, errors.New("this endpoint is not implemented yet"))
 }
 
 // DeleteEmergencyRoom godoc
-// @Summary      delete an emergency room by id
-// @Tags 		 emergency-rooms
-// @Produce 	 json
-// @Param        authorization  header  string      true    "Bearer: <TOKEN>"
-// @Param        id   path 		string  			true	"Emergency Room's ID"
-// @Success      200  {object} 	StatusResponse
-// @Failure      501  {object}  HTTPErrorResponse
-// @Router       /er/{id} [delete]
+// @Summary    delete an emergency room by id
+// @Tags       emergency-rooms
+// @Produce    json
+// @Param      authorization              header       string                true    "Bearer: <TOKEN>"
```

Comment:
Missing spaces here. Look at line 100 for example. Be consistent here, please!

Reply:
spacing seems fine for me? (both on gh and in IDE)